kickstarter/android-oss
# Sign up or log in from Pledge screen

# What ❓
Adds continue button to pledge fragment so users can log in or sign up during pledge flow.
Tests.

# How to QA? 🤔
As a logged out user, you should see the `Continue` button at the bottom of the pledge screen.
And when you click it, you will be taken to the Login Tout screen with the sign up and log in buttons.
After signing up or logging in, the `Continue` button should be gone and your payment methods should display.

As a logged in user, you should **not** see the `Continue` button at the bottom of the pledge screen and your payment methods should display.

# Story 📖
[Trello](https://trello.com/c/uWKbjbhp/1333-android-user-can-login-or-register-from-pledge-screen)

# See 👀
![2019-04-23 14_59_07 2019-04-23 15_00_00](https://user-images.githubusercontent.com/1289295/56611018-c98b3780-65de-11e9-96fb-fc85fcb37099.gif)
